gitlab/resource_gitlab_project_test.go

```diff
@@ -494,6 +496,25 @@ func TestAccGitlabProject_transfer(t *testing.T) {
 	})
 }
 
+func TestAccGitlabProject_IssueMergeRequestTemplates(t *testing.T) {
+	var project gitlab.Project
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGitlabProjectConfigIssueMergeRequestTemplates(rInt),
```

Comment:
Does this need a `SkipFunc: isRunningInCE` since these options are only available in Premium+?

Reply:
Yes, probably. However, I'm not at the same employer anymore and cannot help carry this any further.